Taskfile.yml

```
@@ -379,6 +413,94 @@ tasks:
       - task: ensure-tflite-symlinks
         vars: {LIB_DIR: '{{.TFLITE_LIB_DIR}}', LIB_FILENAME: '{{.LIB_FILENAME}}'}
 
+  # Internal task for building noembed binaries across platforms
+  noembed_build:
+    internal: true
+    deps: [check-tools, check-tensorflow, download-avicommons-data, frontend-build]
+    cmds:
+      - task: download-tflite
+        vars: 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)